bookkeeper-stats-providers/prometheus-metrics-provider/src/main/java/org/apache/bookkeeper/stats/prometheus/PrometheusMetricsProvider.java

```diff
@@ -128,10 +128,17 @@ public void start(Configuration conf) {
         }
 
         // Include standard JVM stats
-        new StandardExports().register(registry);
-        new MemoryPoolsExports().register(registry);
-        new GarbageCollectorExports().register(registry);
-        new ThreadExports().register(registry);
+        try {
+            new StandardExports().register(registry);
+            new MemoryPoolsExports().register(registry);
+            new GarbageCollectorExports().register(registry);
+            new ThreadExports().register(registry);
+        } catch (Exception e) {
```

Comment:
this approach will cause some exports are not registered. for example, if `StandardExports` was registered but no the others, this approach will cause the others being skipped.

Reply:
I would suggest having a common method that catches exception. 

```
public void registerExports(Exports exports, CollectorRegistry registry) {
     try {
            exports.register(registry);
     } catch (Exception e) {
            ....
     }

}
```

Reply:
@sijie Fixed